example/webpack5/dist/index.js

```diff
@@ -99,7 +99,7 @@ __webpack_require__.r(__webpack_exports__);
 
 var ___CSS_LOADER_EXPORT___ = _example_webpack5_node_modules_css_loader_dist_runtime_api_js__WEBPACK_IMPORTED_MODULE_1___default()((_example_webpack5_node_modules_css_loader_dist_runtime_noSourceMaps_js__WEBPACK_IMPORTED_MODULE_0___default()));
 // Module
-___CSS_LOADER_EXPORT___.push([module.id, ".foo {\n  width: 240px;\n}\n\n.bar {\n  width: 160px;\n}\n\n.header-main {\n  background: red;\n}\n\n.footer-main {\n  background: blue;\n}\n\n@media screen and (min-width: 768px) {\n  .foo {\n    width: 576px;\n  }\n  .bar {\n    width: 384px;\n  }\n}\n\n@media all and (-webkit-min-device-pixel-ratio: 1.5),\n  (min--moz-device-pixel-ratio: 1.5),\n  (-o-min-device-pixel-ratio: 1.5/1),\n  (min-device-pixel-ratio: 1.5),\n  (min-resolution: 138dpi),\n  (min-resolution: 1.5dppx) {\n  .header-main {\n    background: gold;\n    -webkit-background-size: auto auto;\n    -moz-background-size: auto auto;\n    background-size: auto auto;\n  }\n  .footer-main {\n    background: pink;\n    -webkit-background-size: auto auto;\n    -moz-background-size: auto auto;\n    background-size: auto auto;\n  }\n}\n", ""]);
+___CSS_LOADER_EXPORT___.push([module.id, ".foo {\r\n  width: 240px;\r\n}\r\n\r\n.bar {\r\n  width: 160px;\r\n}\r\n\r\n.header-main {\r\n  background: red;\r\n}\r\n\r\n.footer-main {\r\n  background: blue;\r\n}\r\n\r\n@media screen and (min-width: 768px) {\r\n  .foo {\r\n    width: 576px;\r\n  }\r\n  .bar {\r\n    width: 384px;\r\n  }\r\n}\r\n\r\n@media all and (-webkit-min-device-pixel-ratio: 1.5),\r\n  (min--moz-device-pixel-ratio: 1.5),\r\n  (-o-min-device-pixel-ratio: 1.5/1),\r\n  (min-device-pixel-ratio: 1.5),\r\n  (min-resolution: 138dpi),\r\n  (min-resolution: 1.5dppx) {\r\n  .header-main {\r\n    background: gold;\r\n    -webkit-background-size: auto auto;\r\n    -moz-background-size: auto auto;\r\n    background-size: auto auto;\r\n  }\r\n  .footer-main {\r\n    background: pink;\r\n    -webkit-background-size: auto auto;\r\n    -moz-background-size: auto auto;\r\n    background-size: auto auto;\r\n  }\r\n}\r\n", ""]);
```

Comment:
Seem on you operation system (windows) used `r\n` instead `\n` to brake lines 

Could you please discard these changes?

```
git checkout origin/master -- example/webpack5/dist/index.js
```

Reply:
nothing changes when I write this command, what am I doing wrong?

![image](https://user-images.githubusercontent.com/79831859/134543556-69b244ab-2f25-4527-85ad-784d2f2c56fc.png)

Reply:
```
# ssh
git remote add base git@github.com:retyui/group-css-media-queries-loader.git
# or https
git remote add base https://github.com/retyui/group-css-media-queries-loader.git

# then try
git checkout base/master -- example/webpack5/dist/index.js example/webpack5/webpack.config.ts
```